package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@mtfh/property",
   "version": "0.0.1",
   "scripts": {
-    "start": "webpack serve --port 8010",
+    "start": "SET NODE_OPTIONS=--openssl-legacy-provider && webpack serve --port 8010",
```

Comment:
What is this option set for?

Reply:
It's added to prevent this error from occurring:

`{
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}`